drafts/2019-10-22-liberty-dev-mode.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Open Liberty development mode"
+categories: blog
+author_picture: https://avatars2.githubusercontent.com/u/29640130
+author_github: https://github.com/ericglau
+seo-title: Open Liberty development mode - OpenLiberty.io
+seo-description: Development mode for Open Liberty allows you to develop applications with any text editor or IDE by providing hot reload and deployment, on demand testing, and debugger support.
+blog_description: "Development mode for Open Liberty allows you to develop applications with any text editor or IDE by providing hot reload and deployment, on demand testing, and debugger support."
+---
+= Open Liberty development mode
+Eric Lau <https://github.com/ericglau>
+
+Open Liberty development mode, or dev mode, allows you to develop your application with any text editor or IDE.  Your code will be automatically compiled and deployed to your running server, making it easy to iterate on your changes. You will be able to run tests on demand or even automatically so that you can get immediate feedback on your changes. You will also be able to attach a debugger at any time to debug your running application.
```

Comment:
Replace the first sentence with something like the sentence you used in the metadata, eg "Development mode for Open Liberty allows you to develop applications with any text editor or IDE by providing hot reload and deployment, on demand testing, and debugger support.". The current first sentence just says you can develop your app in any editor/IDE which is not very compelling...

Reply:
Use present tense - easier to read than future tense; eg "you code is automatically compiled" instead of "your code will be automatically compiled".